cucoslib/dispatcher/nodes.yml

```diff
@@ -325,6 +335,16 @@
         encryption: false
         versioned: true
 
+    - name: 'S3GitHubManifestMetadata'
+      import: 'cucoslib.storages'
+      configuration:
+        aws_access_key_id: 'GNV3SAHAHA3DOT99GQII'
+        aws_secret_access_key: 'ZmvMwngonaDK5ymlCd6ptaalDdJsCn3aSSxASPaZ'
```

Comment:
Based on https://github.com/fabric8-analytics/fabric8-analytics-worker/pull/27 please replace this with `<<: *configurationS3` to avoid copy-paste errors.